sitemap.br.xml

```diff
@@ -1,5 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url>
+    <loc>https://blog.adobe.com/br/publish/2021/04/08/por-que-ser-um-cidadao-global-aumenta-a-criatividade.html</loc>
+    <lastmod>2021-04-08</lastmod>
+  </url>
+  <url>
+    <loc>https://blog.adobe.com/br/publish/2021/03/28/como-ter-um-curriculo-vitae-criativo-e-impressionante-modelos-gratuitos.html</loc>
+    <lastmod>2021-03-28</lastmod>
+  </url>
+  <url>
+    <loc>https://blog.adobe.com/br/publish/2021/02/25/como-ter-um-curriculo-vitae-criativo-e-impressionante-modelos-gratuitos.html</loc>
```

Comment:
Isn't this a duplicate of the entry above?

Reply:
Actually this one doesn't exist anymore... maybe a move that got stuck in the index? @dominique-pfister

Reply:
The eye of the tiger ;)
The BR index was outdated, I re-published it and it fixed this duplicate.

Reply:
Happened to be the first file in the review 😄